src/packet_analysis/Analyzer.cc

```diff
@@ -68,34 +68,60 @@ bool Analyzer::IsAnalyzer(const char* name)
 	return packet_mgr->GetComponentName(tag) == name;
 	}
 
+const AnalyzerPtr& Analyzer::LookupAnalyzer(uint32_t identifier) const
```

Comment:
Naming nit as already discussed: I'd prefer to not reuse the name `LookupAnalyzer` here. Alternatives might be `DoLookup` or `Dispatch`. That being said, I wouldn't mind a breaking change using `Lookup`.

Reply:
Yeah, seems more reasonable to just break the API.

@timwoj , I'm going to push that and move Dispatcher into zeek::packet_analysis::detail. It doesn't look like a user could actually get direct access to a `Dispatcher` instance or put their own implementations in place if they wanted to, so this is a total implementation detail anyhow.

Reply:
> and move Dispatcher into zeek::packet_analysis::detail. 

Actually, nope. That causes a ton of fallout because we have much `detail::ABC` references within the `packet_analysis` namespace and they all break when introducing a `packet_analysis::detail` namespace :sigh:

Reply:
> Naming nit as already discussed: I'd prefer to not reuse the name LookupAnalyzer here. Alternatives might be DoLookup or Dispatch. That being said, I wouldn't mind a breaking change using Lookup.

Ah, sorry Jan. I changed the `Dispatcher::LookupAnalyzer` one back to `Lookup` as that one is definitely internal. I now realize this was for `Analyzer`. Let me mull it over, though `LookupAnalyzer` as the public API seems preferable to the other names. But yes, naming... If the name re-use concern is with the private methods, those being `LookupInner` and `DetectInner` may be an improvement. I mostly came from an public API angle, the rest can always easily be changed.